ViAn/GUI/mainwindow.cpp

```diff
@@ -754,16 +755,22 @@ void MainWindow::remove_video_from_tree(MyQTreeWidgetItem *my_video) {
 
 /**
  * @brief MainWindow::toggle_toolbar
- * This method will toggle the toolbar depending on whether the overlay is showing or not.
+ * This method will toggle the toolbar depending on whether the overlay and analysis is showing or not.
  * It is switching between a toolbar that contains items as save/add/load and another that
- * contains drawing tools.
- * This is invoked when the overlay is activated and deactivated.
+ * contains drawing tools, and a third containing analysis tools.
+ * This is invoked when the overlay and analysis tool is activated and deactivated.
  */
 void MainWindow::toggle_toolbar() {
-    if(mvideo_player->is_showing_overlay()) {
+    if (mvideo_player->is_showing_analysis_tool()) {
+        ui->toolBar_analysis->show();
+        ui->toolBar_no_overlay->hide();
```

Comment:
"toolBar_no_overlay" probably a bad name. If it isnt the standard toolbar, please name it accordingly